libraries/src/AWS.Lambda.Powertools.Logging/Internal/LoggingAspectHandler.cs

```diff
@@ -67,7 +67,7 @@ internal class LoggingAspectHandler : IMethodAspectHandler
     private readonly LoggerOutputCase? _loggerOutputCase;
 
     /// <summary>
-    ///     The Powertools configurations
+    ///     The Powertools for AWS Lambda (.NET) configurations
```

Comment:
I wouldn't change here. comment in code

Reply:
We're unable to use "Powertools" as a standalone brand any more due to internal issues